composer.json

```diff
@@ -33,10 +33,10 @@
     "stellarwp/db": "^1.0.3",
     "stellarwp/installer": "^1.1.0",
     "stellarwp/models": "dev-main",
-    "stellarwp/schema": "^1.1.8",
+    "stellarwp/schema": "^2.0",
+    "stellarwp/shepherd": "0.0.2",
     "stellarwp/telemetry": "^2.3.4",
-    "stellarwp/uplink": "2.2.2",
-    "woocommerce/action-scheduler": "3.9.2"
```

Comment:
If you remove it, this will fail: https://github.com/the-events-calendar/tribe-common/blob/master/src/Tribe/Main.php#L527

Reply:
It won't, because we do the same in Shepherd. But to be clearer i'll remove the requirement in our own codebase too.

Action scheduler will continue to be available through Shepherd.